backend/de.metas.acct.base/src/main/sql/postgresql/ddl/functions/report_taxaccounts.sql

```diff
@@ -0,0 +1,385 @@
+CREATE OR REPLACE FUNCTION de_metas_acct.report_taxaccounts(p_ad_org_id     numeric,
+                                                            p_account_id    numeric,
+                                                            p_c_vat_code_id numeric,
+                                                            p_datefrom      date,
+                                                            p_dateto        date,
+                                                            p_isshowdetails character DEFAULT 'N'::bpchar,
+                                                            p_level         character DEFAULT NULL::bpchar,
```

Comment:
pls document here (on top) 
what's the meaning of `p_level` param. what values can take and what's the meaning of each value